.github/workflows/ci-workflow.yaml

```
@@ -0,0 +1,23 @@
+name: CI Workflow 
+on: [push, pull_request]
+jobs:
+  build:
+    name: build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [11.x]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with: 
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm ci
+      - name: Build next.js
+        run: npm run next:build
+      - name: Run tests
+        run: npm run test
```

```suggestion
        NODE_ENV=test npx jest --coverage --verbose
```

based on https://github.com/freeCodeCamp/chapter/blob/93f6aad27c81a86f655f27ca63d9069a442d06ff/package.json#L21

---

The `npm run test` script, in this case, would be equivalent to the one defined in the package.json hence would resolve to `NODE_ENV=test npx jest --coverage --verbose`

---

Agreed with @Lobarr 👍

---

Cool, got it!